concent_api/utils/logging.py

```diff
@@ -176,34 +199,58 @@ def log_changes_in_subtask_states(client_public_key: bytes, count: int):
     ))
 
 
-def log_change_subtask_state_name(old_state, new_state):
+def log_change_subtask_state_name(
+    logger,
+    old_state,
+    new_state
+):
     logger.info('Subtask changed its state from {} to {}'.format(
         old_state,
         new_state,
     ))
 
 
-def log_new_pending_response(response_type: str, queue_name: str, task_id: str, subtask_id: str, client_public_key: bytes):
-    logger.info('New pending response in {} endpoint -- RESPONSE_TYPE: {} -- TASK_ID: {} -- SUBTASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+def log_new_pending_response(
+    logger,
+    response_type: str,
+    queue_name: str,
+    subtask: Subtask
+):
+    logger.info('New pending response in {} endpoint RESPONSE_TYPE: {} TASK_ID: {} SUBTASK_ID: {} PROVIDER PUBLIC KEY: {} REQUESTOR PUBLIC KEY {}'.format(
         queue_name,
         response_type,
-        task_id,
-        subtask_id,
-        client_public_key,
+        subtask.task_id if subtask is not None else 'UNAVAILABLE',
+        subtask.subtask_id if subtask is not None else 'UNAVAILABLE',
+        subtask.provider.public_key_bytes if subtask is not None else 'UNAVAILABLE',
+        subtask.requestor.public_key_bytes if subtask is not None else 'UNAVAILABLE',
     ))
 
 
-def log_receive_message_from_database(message: Message, client_public_key: bytes, response_type: str, queue_name: str):
-    logger.info('Message {}, TYPE: {} has been received by {} endpoint -- RESPONSE_TYPE: {} -- CLIENT PUBLIC KEY: {}'.format(
+def log_receive_message_from_database(
+    logger,
+    message: Message,
+    client_public_key: bytes,
+    response_type: str,
+    queue_name: str
+):
+    logger.info('Message {}, TYPE: {} has been received by {} endpoint. RESPONSE_TYPE: {} TASK_ID: {} SUBTASK_ID: {} CLIENT PUBLIC KEY: {}'.format(
```

Comment:
Please use f-strings

Reply:
Ok, I will do it, but after all other changes, in separate commit. I thing it will be better to read- one commit with 'logic' changes and one with style. Is it OK?

Reply:
If you are changing some strings or adding new ones you should do it correctly at once. If there is left any more strings which are using `format()` function and you don't change anything there, you can do it in another commit. It's good idea if you are working with this file to change all strings which are not f-string in new commit, but remember not to do same job two twice.

Reply:
Actually, changing to f-strings in a separate commit is a good idea. Same applies to wrapping parameter lists. If the logic changs are in separate commits, it's much easier to review.